internal/jet/table.go

```diff
@@ -186,3 +186,68 @@ func (t *joinTableImpl) serialize(statement StatementType, out *SQLBuilder, opti
 		t.onCondition.serialize(statement, out)
 	}
 }
+
```

Comment:
I don't understand. Whats the point of the changes in this file? 
Lateral is not a join type. Lateral logic is already implemented in the changes above (`lateralTableImpl`).

Reply:
To be used as shortcut, it can be deleted, please refer to below answer.